infra/resources/_modules/backend_func/apim_platform_api.tf

```diff
@@ -0,0 +1,67 @@
+resource "azurerm_api_management_product" "apim_platform_cdc_product" {
+  product_id   = "io-cdc-public-api"
+  display_name = "IO CDC PUBLIC API"
+  description  = "Product for IO CDC"
+
+  api_management_name = data.azurerm_api_management.apim_platform.name
+  resource_group_name = data.azurerm_api_management.apim_platform.resource_group_name
+
+  published             = true
+  subscription_required = false
+  approval_required     = false
+}
+
+resource "azurerm_api_management_product_policy" "apim_platform_cdc_product_policy" {
+  product_id          = azurerm_api_management_product.apim_platform_cdc_product.product_id
+  api_management_name = azurerm_api_management_product.apim_platform_cdc_product.api_management_name
+  resource_group_name = azurerm_api_management_product.apim_platform_cdc_product.resource_group_name
+
+  xml_content = file("${path.module}/api_product/_base_policy.xml")
+}
+
+resource "azurerm_api_management_api_version_set" "apim_platform_cdc_api_version_set" {
```

Comment:
Andrea suggest me to use simpler naming convention for the resources describing what it is.
In this case for example only `cdc`. When you refer this resource use `azurerm_api_management_api_version_set.cdc` (api_version_set and apim are implicit on the resource and platform make sense only if you use more then one apim). Apply this principle even on the other resources

Reply:
done https://github.com/pagopa/io-cdc/pull/12#:~:text=fixed%20review-,e974311,-Merge%20branch%20%27iobp